juniper_official/routing/collect-rsvp-stats.rule

```diff
@@ -0,0 +1,200 @@
+/*
+ * Collect RSVP LSP traffic statistics for pathfinder dynamic topology.
+ *
+ * Req 60: https://paragon-automation.atlassian.net/browse/REQ-60
+ *
+ * One input control detection
+ *
+ *   1) input-lsp-name, is a regular expression that matches the lsp name
+ *      that you would like to monitor.  By default it's '.*', which matches
+ *      all lsps. Use something like 'Silver*' to match only lsp names starting
+ *      with 'Silver'. 
+ *
+ */
+ healthbot {
+    topic routing.mpls {
+        rule collect-rsvp-stats {
```

Comment:
What is the scale that we targeting for this rule?

Reply:
Response from Reinhard: I haven't found any number of required LSPs to be supported. In Northstar, we have customers who have 100k + lsps to be managed, but the bulk has less than 100k.
  Having said that, I would go with 100k for now. This is not per device, but for the entire network. I believe this number will only be relevant if existing Northstar customers upgrade to EOP.